### PR TITLE
Add min_distance field in ccd.

### PR DESCRIPTION
**Pull request by arvigj**
Added optional min_distance argument to `is_step_collision_free` and `compute_collision_free_stepsize`.

### CODE REVIEW

File: src/ipc/candidates/edge_edge.cpp [hunk 1]
```diff
@@ -60,6 +60,7 @@ bool EdgeEdgeCandidate::ccd(
     const Eigen::MatrixXi& E,
     const Eigen::MatrixXi& F,
     double& toi,
+    const double min_distance,
```
Comment (zfergus) [inline src/ipc/candidates/edge_edge.cpp:63]: doesn't look like you are using this parameter

File: src/ipc/candidates/continuous_collision_candidate.hpp [hunk 2]
```diff
@@ -29,6 +29,7 @@ struct ContinuousCollisionCandidate {
         const Eigen::MatrixXi& E,
         const Eigen::MatrixXi& F,
         double& toi,
+        const double min_distance = 0.0,
```
Comment (zfergus) [inline src/ipc/candidates/continuous_collision_candidate.hpp:32]: can you add a Doxygen comment for the parameter?

File: src/ipc/candidates/face_vertex.cpp [hunk 3]
```diff
@@ -56,6 +56,7 @@ bool FaceVertexCandidate::ccd(
     const Eigen::MatrixXi& E,
     const Eigen::MatrixXi& F,
     double& toi,
+    const double min_distance,
```
Comment (zfergus) [inline src/ipc/candidates/face_vertex.cpp:59]: param not used

File: src/ipc/ipc.cpp [hunk 4]
```diff
@@ -214,6 +214,7 @@ bool is_step_collision_free(
     const Eigen::MatrixXd& V0,
     const Eigen::MatrixXd& V1,
     const BroadPhaseMethod method,
+    const double min_distance,
```
Comment (zfergus) [inline src/ipc/ipc.cpp:217]: you should also set the inflation radius in `construct_collision_candidates` to `min_distance/2`

File: src/ipc/ipc.cpp [hunk 5]
```diff
@@ -265,6 +268,7 @@ double compute_collision_free_stepsize(
...
```
Comment (zfergus) [inline src/ipc/ipc.cpp:271]: use `min_distance/2` for inflation radius in `construct_collision_candidates` call

**Review by zfergus — CHANGES_REQUESTED** [review]

File: src/ipc/candidates/face_vertex.cpp (hunk 3 above)
Comment (arvigj) [reply src/ipc/candidates/face_vertex.cpp:59]: Should be added to call below, fixed.

File: src/ipc/candidates/edge_edge.cpp (hunk 1 above)
Comment (arvigj) [reply src/ipc/candidates/edge_edge.cpp:63]: Should be added to call below, fixed.

**Review by zfergus — APPROVED** [review]

**Pull request merged**